pkg/workflows/workflows_test.go

```diff
@@ -0,0 +1,107 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package workflows
```

Comment:
BTW with the changed code organisation structure, we can rewrite the `Makefile` code coverage testing command from:

```
go test -cover `go list ./... | grep -v /reana-client-go/client`
```

to use the three leading leading directories that we have introduced:

```
go test -cover ./cmd/... ./config/... ./pkg/...
```